src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -85,7 +85,7 @@ public function block($messages, $type = null, $style = null, $prefix = ' ', $pa
 
         foreach ($lines as &$line) {
             $line = sprintf('%s%s', $prefix, $line);
-            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line));
+            $line .= str_repeat(' ', $this->lineLength - Helper::strlen($line) + substr_count($line, '\<'));
```

Comment:
https://github.com/symfony/symfony/pull/14600/files#diff-18b76c26b226bea37b2b821a5e4962ffR88

Let me know what comment you expect if you think it's better :)